src/sagas.js

```diff
@@ -700,6 +697,11 @@ function* schedulerPerMinute (action) {
   }
 }
 
+function* delayedRemoveActiveRequest ({ payload: requestId }) {
```

Comment:
I think this action doesn't give any sense, because chance, that next request will be in 1 ms after `removeActiveRequest`, the same as if next request will be in 501 ms.